src/pages/Proposal/ProposalSummary/Stepper/index.tsx

```diff
@@ -0,0 +1,159 @@
+/** @jsxImportSource @emotion/react */
+import React, { useMemo } from 'react';
+import { Typography } from '@mui/material';
+import { Icon } from 'components';
+import { useTranslation } from 'translation';
+import { IProposal } from 'types';
+import { useStyles } from './styles';
+
+export interface IStepperProps {
+  className?: string;
+  proposal: IProposal;
+}
+
+const StepStates = {
+  Pending: { active: 1, error: undefined },
+  Active: { active: 2, error: undefined },
+
+  Defeated: { active: 3, error: true },
+  Succeeded: { active: 3, error: undefined },
+  Canceled: { active: 3, error: true },
+  Expired: { active: 3, error: true },
+
+  Queued: { active: 4, error: undefined },
+
+  Executed: { active: 5, error: undefined },
+};
```

Comment:
It looks like `error` is never used, so you could use a function that returns a step index based on a proposal state instead:
```suggestion
const getStepIndex = (proposalState: ProposalState) => {
  switch (proposalState) {
    case 'Pending':
      return 1;
    case 'Active':
      return 2;
    case 'Defeated':
    case 'Succeeded':
    case 'Canceled':
    case 'Expired':
      return 3;
    case 'Queued':
      return 3;
    default:
      return 5;
  }
};
```

Reply:
ahh yeah that was left over from trying to use material ui. I'll update this